cpp/velox/CMakeLists.txt

```diff
@@ -582,9 +265,71 @@ if(BUILD_JEMALLOC)
 endif()
 
 target_link_libraries(velox PUBLIC gluten)
-add_velox_dependencies()
+
+add_library(facebook::velox STATIC IMPORTED)
+set_target_properties(
+  facebook::velox PROPERTIES IMPORTED_LOCATION
+                             ${VELOX_BUILD_PATH}/lib/libvelox.a)
+
+if(BUILD_TESTS)
+  add_library(facebook::velox::dbgen STATIC IMPORTED)
```

Comment:
@PHILO-HE For the testing libraries, they are still separate libraries, right?

Reply:
@xumingming, yes, these test libs are still separate.